common/changes/@itwin/certa/nam-core-frontend-vitest_2024-10-04-14-51.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/certa",
+      "comment": "",
```

Comment:
for this, itd be worth mentioning updating playwright version

Reply:
done